src/common/textures/bitmap.cpp

```diff
@@ -257,21 +258,21 @@ bool ClipCopyPixelRect(const FClipRect *cr, int &originx, int &originy,
 		step_y = pstep_y;
 		break;
 
-	case 1: // rotate 90� right
+	case 1: // rotate 90° right
```

Comment:
What's going on here? Why's the text altered like this? cc @coelckers

Reply:
The degree sign is problematic on some editors. I attempted fixing this on Visual Studio Code but seems like it doesn't work correctly. It always saves or displays it as (or with) an unknown symbol.

Reply:
Ok changed it to "degrees".